src/NuGet.Clients/NuGet.PackageManagement.VisualStudio/Telemetry/RestoreBannerClickedTelemetryEvent.cs

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Common;
+
+namespace NuGet.PackageManagement.Telemetry
+{
+    public sealed class RestoreBannerClickedTelemetryEvent : TelemetryEvent
+    {
+        internal const string RestoreBannerClickedEventName = "RestoreBannerClicked";
```

Comment:
I would add more event details. For example: ?

In which tab the user is seeing the banner? Installed, Update,  Browse (if applicable)
Is the benner visible in Solution PM UI ? If so, could we add a field to indicate if user is seeing the banner either in project or solution PM UI ?

Reply:
The banner will show in all the tabs, so I think we can skip that